book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accesor :title, :author
```

Comment:
- [ ] Kindly make sure to add the `rentals` to `attr_accessor` so it can be accessible

Reply:
Well resolved 👍🏽